bundles/org.openhab.binding.evcc/README.md

```diff
@@ -1,257 +1,58 @@
-| [loadpoint\<N\>\|vehicle\<ID\]>#vehicleLimitSoC    | Number:Dimensionless | RW         | Until which state of charge (SoC) should the specific vehicle be charged |
-| [loadpoint\<N\>\|vehicle\<ID\]>#vehicleCapacity    | Number:Energy        | R          | Capacity of EV battery                                                   |
-| [loadpoint\<N\>\|vehicle\<ID\]>#vehiclePlanEnabled | Switch               | RW         | Plan for charging enabled                                                |
-| [loadpoint\<N\>\|vehicle\<ID\]>#vehiclePlanSoC     | Number:Dimensionless | RW         | Until which state of charge (SoC) should vehicle be charged in plan      |
-| [loadpoint\<N\>\|vehicle\<ID\]>#vehiclePlanTime    | DateTime             | RW         | When the plan SoC should be reached                                      |
-
-### Heating Channels
-
-Those channels exist:
-
-- 1 per configured loadpoint with `chargerFeatureHeating = true`:
-  - These channels point to the heating device that is currently active/connected at/to the loadpoint
-  - Please note that you have to replace _\<N\>_ with your loadpoint id/number
-- 1 per configured heating device:
-  - Please note that you have to replace _\<ID\>_ with your heating device id/name
-
-| Channel                                                 | Type               | Read/Write | Description                                                           |
-|---------------------------------------------------------|--------------------|------------|-----------------------------------------------------------------------|
-| [loadpoint\<N\>\|heating\<ID\]>#heatingTitle            | String             | R          | Title of heating device                                               |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingMinTemperature   | Number:Temperature | RW         | Minimum Temperature a heating device should have                      |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingLimitTemperature | Number:Temperature | RW         | Until which Temperature should the specific heating device be charged |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingCapacity         | Number:Energy      | R          | Capacity of heating device                                            |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingPlanEnabled      | Switch             | RW         | Plan for charging enabled                                             |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingPlanTemperature  | Number:Temperature | RW         | Until which Temperature should heating device be charged in plan      |
-| [loadpoint\<N\>\|heating\<ID\]>#heatingPlanTime         | DateTime           | RW         | When the plan Temperature should be reached                           |
+Channels will be created dynamically
 
 ## Full Example
 
 ### Thing(s)
 
 ```java
-Thing evcc:device:demo "evcc Demo" [url="https://demo.evcc.io", refreshInterval=60]
-```
-
-### Items
```

Comment:
Even when all is dynamically, please create a (subset) as it will be helpfull for those who define channel / item links in a file based setup.